Track.h

```diff
@@ -109,12 +127,20 @@ class Track
 private:
   TrackState state_;
   HitVec hits_;
-  HitVec initHits_;
-  HitIdxVec hitIdxVec_;
   float chi2_;
+  unsigned int mcTrackID_;
+  unsigned int nHitsMatched_;
+  unsigned int seedID_;
+  unsigned int duplicateID_;
+  bool isDuplicate_;
+  HitIdxVec hitIdxVec_;
```

Comment:
Hi @cerati , yes, this branch is all ready to go, except for unsigned int -> int.  To reiterate, I still need some guidance on what to do with them in Geometry / binning.